### DISCUSSION TITLE
Extension not found

### DISCUSSION THREAD

**Discussion by benoitseron** [community/community#40996] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I cannot find the extension in the extension manager, nor install it through doing Ctrl+P and running ext install GitHub.copilot.

I am using "code" on Garuda linux.

Version: 1.72.1
Commit: 129500ee4c8ab7263461ffe327268ba56b9f210d
Date: 2022-10-13T20:17:28.261Z
Electron: 19.0.13
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 6.0.2-zen1-1-zen
Sandboxed: No

**Comment by ossanna16** [comment]
Hi there @benoitseron and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
